apps/src/studio/api.js

```diff
@@ -166,6 +166,14 @@ exports.changeScore = function (id, value) {
   Studio.queueCmd(id, 'changeScore', {'value': value});
 };
 
+exports.getScore = function () {
+  return Studio.playerScore;
+};
+
+exports.setScore = function (value) {
+  Studio.setScore(value);
```

Comment:
Yes I believe this is correct.  This may be a hassle when we move to non-stale variables for queued commands, but it's how all other variable declarations work today.